receiver/zipkin/trace_receiver_test.go

```diff
@@ -156,11 +156,7 @@ func TestConversionRoundtrip(t *testing.T) {
   }
 }]`)
 
-	sink := new(noopSink)
-	zi, err := New(sink)
-	if err != nil {
-		t.Fatalf("Failed to create the Zipkin receiver: %v", err)
-	}
+	zi := &ZipkinReceiver{spanSink: new(noopSink)}
```

Comment:
Why not zr?

Reply:
Sure, I can rename that, that code was just moved then `spanSink` added. Thanks.